docs/kf.dev/content/en/docs/operators/administering-kf/setting-up-a-custom-domain.md

```diff
@@ -0,0 +1,64 @@
+---
+title: "Setting up a custom domain"
+weight: 55
+type: "docs"
+---
+
+By default, Kf apps are accessible on domains exposed by Knative Serving.
+Kf extends Knative Serving's domain system to allow path based routing, per-app domains, wildcard domains, and other routing services.
+
+## Set up a domain
+
+Follow the [Knative Serving](https://knative.dev/docs/serving/using-a-custom-domain/) docs to set up a custom domain for your cluster.
+
+## Assign domains to spaces
+
+After you've set up a custom domain for your cluster, you can assign domains and sub-domains to each space for developers to use.
```

Comment:
In theory, you don't have to setup knative with the domain first. Do we care about that at all?

Reply:
Clarified wording around that.